.circleci/config.yml

```diff
@@ -170,7 +169,7 @@ jobs:
     steps:
       - install-bazel-linux
       - checkout
-      - run: bazel test //:release-validate-deps --test_output=streamed
+      - run: bazel test //:release-validate-deps
```

Comment:
let's add `--test_output=streamed`

Reply:
resolved in https://github.com/graknlabs/workbase/pull/342/commits/056eb50959e086984fb642c24f07019e00dfb1d2